packages/app/src/containers/TextStyleInspector.tsx

```diff
@@ -0,0 +1,220 @@
+import Sketch from '@sketch-hq/sketch-file-format-ts';
+import { Divider } from 'noya-designsystem';
+import { Selectors } from 'noya-state';
+import { useCallback, memo, useMemo } from 'react';
+import useShallowArray from '../hooks/useShallowArray';
+import TextOptionsRow from '../components/inspector/TextOptionsRow';
+import TextAlignmentRow from '../components/inspector/TextAlignmentRow';
+import TextStyleRow from '../components/inspector/TextStyleRow';
+import {
+  useSelector,
+  useApplicationState,
+} from '../contexts/ApplicationStateContext';
+
+/**
+ * TODO: Handle mixed styles
+ * TODO: Maybe some of this style to a selector
+ */
+
+export default memo(function TextStyleInspector() {
+  const [, dispatch] = useApplicationState();
+
+  //Todo: Handle multiple texts
+  const selectedTextStyles = useShallowArray(
+    useSelector(Selectors.getSelectedTextStyles),
+  );
+
+  const seletedText = useShallowArray(useSelector(Selectors.getSelectedText));
+
+  const fontColor = useMemo(
```

Comment:
I think we can combine most of these into a single `useMemo`. When any property of the text style changes, we're going to have to re-render almost everything anyway, so it shouldn't make a difference in terms of performance.

Something like the `getTextStyleAttributes` function I added to `SketchText.tsx`

Reply:
Oh that's a cool and cleaner solution 👍